src/test/scala/uk/gov/homeoffice/cirium/FlightScheduledRoutesSpec.scala

```diff
@@ -44,6 +46,16 @@ class FlightScheduledRoutesSpec extends Specification with FlightScheduledRoutes
     }
 
   }
+
+  "Given json string without request object" should {
```

Comment:
Shall we remove this test as `CiriumScheduledResponse` doesn't have a request property?